### PR TITLE
feat: landing update

### PR DESCRIPTION
**Pull request by rafaelderolez**
## 📑 Description

This PR is a suggestion to improve the landing on astronvim.com.

- [x] Removed Splash and Starfield.
- [x] Added github url to header icons.
- [x] Removed "Read Source" button, as it takes away from the main CTA (Get Started). I would also argue that people who are interested in neovim distro's are specifically _not_ interested in reading the source code. I just want something that works without me having to understand the core of the setup.
- [x] Responsiveness in the ContentSection has been improved a bit to work better on smaller screens.
- [x] Added a larger AstroNvim image in the intro.
- [x] Reduced the AstroNvim logo in the header just slightly.

### Before
Capture-2023-10-20-095741

### After
Capture-2023-10-20-095712

### CODE REVIEW

File: src/components/header.astro [hunk 1]
```diff
@@ -8,21 +8,26 @@ const navItems: Array<NavItem> = [
...
 >
   <a
     class="flex items-center gap-3 hover:!text-default"
     href="#"
     aria-label="AstroNvim"
   >
-    <img src="/logo/astronvim.svg" alt="AstroNvim logo" class="h-10" />
-    <h1 class="hidden h-6 font-bold md:block">AstroNvim</h1>
+    <img src="/logo/astronvim.svg" alt="AstroNvim logo" class="h-5" />
+    <h1 class="hidden h-4 font-bold leading-none md:block">AstroNvim</h1>
```
Comment (Infonautica) [inline src/components/header.astro:30]: Why is `h1` hidden?
Wondering wouldn't it be better to have title next to a logo to improve "brand recognition" by strengthening association? Also could make it a bit more balanced visually

**Comment by mehalter** [comment]
@rafaelderolez there is a bug where the sticky header on Mobile is not accessible when you scroll past the image.  You clhave to scroll all the way back up to the top above the image

File: src/components/header.astro (hunk 1 above)
Comment (rafaelderolez) [reply src/components/header.astro:30]: @Infonautica The h1 is currently hidden on mobile and shown as soon as you hit the `md` breakpoint. I did not change this behaviour, I only touched its size and alignment.

**Comment by rafaelderolez** [comment]
> @rafaelderolez there is a bug where the sticky navbar on Mobile is not accessible when you scroll past the image. You clhave to scroll all the way back up to the top above the image

@mehalter Ah weird, I'll take a look at that ✌️

File: src/components/header.astro (hunk 1 above)
Comment (mehalter) [reply src/components/header.astro:30]: Yeah, this is intended. It's too large for smaller screens so we hide it.

**Comment by mehalter** [comment]
@rafaelderolez Thanks for these changes! I really dig them 😄  I made  a few commits to do a few things:

1. do some cleanup and remove more (now) unnecessary files
2. make the logo in the nav menu on small screens clickable to go to the top of the page
3. make the hero image even larger and decrease overall y padding

Let me know what you think of the new test deployment and if you have any questions/concerns! This is looking awesome!

**Comment by mehalter** [comment]
Approved by @kabinspace so let's merge! Thanks @rafaelderolez for the heavy lifting on this ❤️

**Pull request merged**